Gopkg.lock

```diff
@@ -1067,6 +1067,7 @@
     "github.com/docker/docker/client",
     "github.com/go-macaron/binding",
     "github.com/gocql/gocql",
+    "github.com/golang/snappy",
```

Comment:
huh? what is this doing here

Reply:
The checks were failing so I needed to run `dep ensure` and that was the change.